app/views/bikes/_bike.html.haml

```diff
@@ -6,18 +6,17 @@
       %h5.title-link
         %a{ href: bike_url(bike) }
           %strong
-            - unless bike.type == 'bike'
-              = bike.type
+            = bike.type unless bike.type == 'bike'
             = [bike.year, bike.mnfg_name].reject(&:blank?).join(' ')
-          #{bike.frame_model}
+          = bike.frame_model.to_s
       %ul.attr-list
-        = bike.attr_list_item(bike.serial_display, 'Serial')
-        = bike.attr_list_item(bike.frame_colors.to_sentence, 'Primary colors')
+        = bike.attr_list_item(bike.serial_display, t(".serial"))
+        = bike.attr_list_item(bike.frame_colors.to_sentence, t(".primary_colors"))
       - if bike.current_stolen_record.present?
         %ul.attr-list
           %li
             %span.attr-title
-              = bike.recovered ? "Recovered" : "Stolen"
+              = bike.recovered ? t(".recovered") : t(".stolen")
```

Comment:
We do this a lot. Should we translate this in the model?

Reply:
Or maybe it should be in a helper?

Reply:
I think this should use the same thing we use for the other model level strings - like the names in `CycleType` and `Color`

Reply:
A helper makes sense, I'll circle back to that. I'll do some research re CycleType and Color.